src/apis/ViaCEP.js

```diff
@@ -0,0 +1,22 @@
+const { APIWrapper } = require('../')
+const fetch = require('node-fetch')
+
+const API_URL = 'https://viacep.com.br/ws/'
+
+module.exports = class ViaCEP extends APIWrapper {
+  constructor () {
+    super({
+      name: 'viacep'
+    })
+  }
+
+  async searchCEP (cep) {
+          return this.request(cep)
```

Comment:
identação ta como

Reply:
I CLICKED APPROVE BY MISTAKE

Reply:
da uma arrumada ai, to resolvendo os conflito